Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -306,6 +306,9 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
     func show(homepanelSection: Route.HomepanelSection) {
         showLibrary(with: homepanelSection)
     }
+    
+    func showEnhancedTrackingProtection() {
+    }
```

Comment:
With this task we can add the ETP coordinator in that function. To be tested further down the line, but I think we can use the same navigation controller from the browser coordinator:
`let libraryRouter = DefaultRouter(navigationController: router.navigationController)`

Then we can add as a child, and call `start` on it. The start method in the ETP coordinator will do nothing for now but will be done later.